### PR TITLE
Updated the roadmap to reflect in progress work

### PR DESCRIPTION
**Pull request by cathysull**
Updated the roadmap with our in progress work and links to those issues.

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
-- Update non-C# source like cshtml, xaml, vb, etc.
-- Post-migration details with all the changes made and brings attention to any potentially breaking changes
-- Even more ideas listed in the repo [issues](https://github.com/dotnet/upgrade-assistant/issues)
+- **Committed**
+  - [Choose between LTS (long-term servicing), Current (.NET 5) or Preview (.NET 6 Preview) for the target .NET version](https://github.com/dotnet/upgrade-assistant/issues/41)
+  - [Add more ASP.NET app analyzers for code fixes](https://github.com/dotnet/upgrade-assistant/issues/55)
+  - Update non-C# source:
+    - [VB](https://github.com/dotnet/upgrade-assistant/issues/270)
+    - [Cshtml](https://github.com/dotnet/upgrade-assistant/issues/57)
+  - Web Forms support
```
Comment (mjrousos) [inline docs/roadmap.md:26]: What's planned for Web Forms? I know we had talked about WebForms -> Blazor upgrade but I feel like that's still in the "idea" stage enough that I wouldn't call it committed.

**Review by mjrousos — APPROVED** [review]
Looks good. Left one comment inline about WebForms stuff that I thought was still further out.

File: docs/roadmap.md [hunk 2]
```diff
@@ -1,28 +1,36 @@
...
-Example areas:
-- Up front checks for technologies that might not be supported in .NET 5+ or could make the upgrade effort large in cost (links to docs, options available, best practices, etc.).
-- More visibility into NuGet dependencies (e.g. newer versions, do they support .NET 5+?)
-- Recommendations for TFM (target frameworks) to move to
+Take a look at the design doc for [design details](https://github.com/dotnet/upgrade-assistant/blob/main/docs/design/Pre-UpgradeAnalysis.md) and phases of implementation.
+
+- **Committed**
+  - [Loose assemblies analysis](https://github.com/dotnet/upgrade-assistant/blob/main/docs/design/Pre-UpgradeAnalysis.md#loose-assembly-analysis)
+- **On Deck**
+  -	More visibility into [NuGet dependencies](https://github.com/dotnet/upgrade-assistant/blob/main/docs/design/Pre-UpgradeAnalysis.md#nuget-package-dependecy-analysis) (e.g. newer versions, do they support .NET 5+?), transitive dependencies, and [inter-project dependencies](https://github.com/dotnet/upgrade-assistant/blob/main/docs/design/Pre-UpgradeAnalysis.md#inter-project-dependencies)
```
Comment (sunandabalu) [inline docs/roadmap.md:14]: This line item is also on the committed bucket

File: docs/roadmap.md [hunk 3]
```diff
@@ -1,28 +1,36 @@
...
+  -	[Up front checks for technologies that might not be supported in .NET 5+](https://github.com/dotnet/upgrade-assistant/blob/main/docs/design/Pre-UpgradeAnalysis.md#surface-unsupported-api-categories) or could make the upgrade effort large in cost (links to docs, options available, best practices, etc.).
+  -	Recommendations for TFM (target frameworks) to move to
```
Comment (sunandabalu) [inline docs/roadmap.md:16]: This is most likely axed, so remove it from the list for now.

File: docs/roadmap.md (hunk 1 above)
Comment (sunandabalu) [reply docs/roadmap.md:26]: Yep, Webforms should be on the on deck pile

File: docs/roadmap.md [hunk 4]
```diff
@@ -1,28 +1,36 @@
...
-- Even more ideas listed in the repo [issues](https://github.com/dotnet/upgrade-assistant/issues)
+- **Committed**
+  - [Choose between LTS (long-term servicing), Current (.NET 5) or Preview (.NET 6 Preview) for the target .NET version](https://github.com/dotnet/upgrade-assistant/issues/41)
+  - [Add more ASP.NET app analyzers for code fixes](https://github.com/dotnet/upgrade-assistant/issues/55)
+  - Update non-C# source:
+    - [VB](https://github.com/dotnet/upgrade-assistant/issues/270)
+    - [Cshtml](https://github.com/dotnet/upgrade-assistant/issues/57)
+  - Web Forms support
+  - Ability to update NuGet dependencies update across a solution instead of project at a time to avoid conflicting cross-dependencies
+  - Multitargeting support for specific and multiple target frameworks
```
Comment (sunandabalu) [inline docs/roadmap.md:28]: I would put multi targeting on deck and not on committed for now.

Comment (mjrousos) [reply docs/roadmap.md:28]: Agreed

File: docs/roadmap.md (hunk 1 above)
Comment (cathysull) [reply docs/roadmap.md:26]: Moved it into the on deck category. Good catch!

File: docs/roadmap.md (hunk 2 above)
Comment (cathysull) [reply docs/roadmap.md:14]: Moved into the committed bucket.

File: docs/roadmap.md (hunk 3 above)
Comment (cathysull) [reply docs/roadmap.md:16]: Removed.

File: docs/roadmap.md (hunk 4 above)
Comment (cathysull) [reply docs/roadmap.md:28]: Done

**Pull request merged**